include/macro_presets.h

```diff
@@ -109,7 +109,7 @@ struct MacroPreset MacroObjectPresets[] = {
     {bhvSmallBully, MODEL_BULLY, 0},
     {bhvSmallBully, MODEL_BULLY_BOSS, 0}, // unused
     {bhvYellowCoin, MODEL_YELLOW_COIN, 0},
-    {bhvStub1D0C, MODEL_UNKNOWN_58, 0}, // unused
+    {bhvYellowCoin, MODEL_UNKNOWN_58, 0}, // unused
```

Comment:
huh.. this is strange. I'd almost recommend keeping the original `bhvStub1D0C` and rename to `bhvUnusedMacro` so people could use it as a custom macro if they _really_ wanted to.

but does anybody use custom macros? i have no clue lol

Reply:
non blocking